lib/fathead/python_six/parse.py

```diff
@@ -26,15 +27,16 @@ def __str__(self):
 
         # Clean up the usage statement which can have newline characters and
         # tab characters, which mess up how it renders 
-
-        code =  '<pre><code>{}</code></pre>'.format(self.usage.replace('\n','\\n')) if self.usage else ''
-       
+        code =  '<pre><code>{}</code></pre>'.format(self.usage.replace('\n','\\n').replace('\t', '    ')) if self.usage else '' 
+         
         # Make the abstract have the description as well as a code block
-        descrip = '<p>'+self.description+'</p>'
-        abstract = '{}{}{}'.format(descrip, '', code)
-        abstract = '<section class="prog__container">' + abstract + '</section>' 
- 
-        return '\t'.join([
+        if code:
+            print self.name
+            abstract = '{}{}'.format(self.description,code)	   
+	else:
+            abstract = self.description
+        abstract = '<section class="prog__container">{}</section>'.format(abstract)  
+        return '    '.join([
```

Comment:
@iamrajhans whoops! You still need to join your 13 fields with `\t`. Those aren't escaped tabs, they're literal tab characters which are necessary for the output.txt to be parseable!